src/js/views/patients/sidebar/action/action-sidebar-forms_views.js

```diff
@@ -102,7 +102,7 @@ const FormLayoutView = View.extend({
       {{#if hasForm}}<h4 class="sidebar__label u-margin--t-8">{{ @intl.patients.sidebar.action.actionSidebarFormsViews.formLayoutView.formLabel }}</h4>{{/if}}
       <div class="flex-grow" data-form-region></div>
     </div>
-    <div class="flex{{#if hasSharing}} u-margin--t-24{{/if}}">
+    <div class="flex">
```

Comment:
is the templateContext still in use?

Reply:
The `hasSharing` value can be removed. But `hasForm` is still in use in that `templateContext()`.